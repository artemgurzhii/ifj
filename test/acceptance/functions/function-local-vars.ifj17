Function function_simple () As Integer
dim b as integer
b = 24
return b
End Function

Scope
dim a as integer
a = function_simple()
End Scope
