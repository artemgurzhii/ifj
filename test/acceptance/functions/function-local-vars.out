.IFJcode17
JUMP Scope
LABEL function_simple
CREATEFRAME
DEFVAR TF@b
MOVE TF@b int@24
PUSHS TF@b
RETURN
LABEL Scope
DEFVAR GF@a
CALL function_simple
POPS GF@a
